stack/__main__.py

```diff
@@ -315,6 +358,7 @@ def add_bucket_permissions(
         member=pulumi.Output.concat('serviceAccount:', WEB_SERVER_SERVICE_ACCOUNT),
     )
 
+    # TODO(@lgruen): remove this once secrets are used for checking memberships.
```

Comment:
Should these be removed now?

Reply:
Not yet, we need to update the analysis-runner / web server first.